hyperliquid/exchange.py

```diff
@@ -293,12 +384,19 @@ def bulk_cancel_by_cloid(self, cancel_requests: List[CancelByCloidRequest]) -> A
         )
 
     def schedule_cancel(self, time: Optional[int]) -> Any:
-        """Schedules a time (in UTC millis) to cancel all open orders. The time must be at least 5 seconds after the current time.
-        Once the time comes, all open orders will be canceled and a trigger count will be incremented. The max number of triggers
-        per day is 10. This trigger count is reset at 00:00 UTC.
+        """
+        Schedules a time (in UTC millis) to cancel all open orders.
 
         Args:
-            time (int): if time is not None, then set the cancel time in the future. If None, then unsets any cancel time in the future.
+            time (Optional[int]): If provided, sets the cancel time in the future (UTC milliseconds).
+                                  If None, unsets any previously scheduled cancel time.
+
+        Note:
+            - The scheduled time must be at least 5 seconds after the current time.
+            - Once the scheduled time arrives, all open orders will be canceled.
```

Comment:
I found having a separate bullet is confusing. The trigger count is incremented when the scheduled time arrives, not when this action is sent.
```suggestion
            - Once the scheduled time arrives, all open orders will be canceled and a trigger count is incremented.
```